devel/nodejs19/Portfile

```diff
@@ -51,16 +48,38 @@ depends_build-append    port:pkgconfig \
 depends_lib-append      path:lib/pkgconfig/icu-uc.pc:icu \
                         port:zlib
 
-# error: 'atomic_load<v8::internal::OwnedVector<const unsigned char> >' is unavailable: introduced in macOS 10.9
```

Comment:
This probably needs a specific compiler choice: at least C/C++ 2011 plus blacklist of a few old Apple clangs.

Likely for gcc this also needs `-latomic` ldflag.